src/redux/reducers/doctor.js

```diff
@@ -0,0 +1,23 @@
+import { FETCH_DOCTOR_SUCCESS, FETCH_DOCTOR_FAILURE, DELETE_DOCTOR } from '../actions/action';
+
+const initialState = [];
+
+const doctorReducer = (state = initialState, { type, payload }) => {
+  switch (type) {
+    case FETCH_DOCTOR_SUCCESS:
+      // cacheDoctors = payload;
```

Comment:
- [ ] Remove this commented line.

Reply:
Thank you @MuriloRoque

Reply:
Hi @MuriloRoque 👋 
Thank you for your review 🙏  In the case of the testing, we literally forgot about it and we do not have enough time to implement them. We couldn't submit this PR and get a review on time due to the bug on the dashboard. Now it's less than 2 hours to the presentation and there is no time to implement the testing. We will ensure to do it in the near future.